CODEOWNERS

```diff
@@ -0,0 +1 @@
+SECURITY.md @Agoric/sec
```

Comment:
GH seem to be complaining about this

Reply:
The permission changes are pending. This group needs write access to repo which will be done before merging.